src/Model.jl

```diff
@@ -55,39 +56,47 @@ discretization information).
 `n_electrons` is taken from `atoms` if not specified
 
 `spin_polarization` is :none by default (paired electrons)
+unless any of the elements has a non-zero initial magnetic moment.
+In this case the spin_polarization will be :collinear.
+
+`magnetic_moments` is only used to determine the symmetry and the
+`spin_polarization`; it is not stored inside the datastructure.
 
 `smearing` is Fermi-Dirac if `temperature` is non-zero, none otherwise
 
-The `symmetry` kwarg can be:
-- `:auto`: determine from terms if they respect the symmetry of the lattice/atoms.
-- `:off`: no symmetries at all
-- `:force`: force all the symmetries of the lattice/atoms.
-Careful that in this last case, wrong results can occur if the
-external potential breaks symmetries (this is not checked).
+The `symmetries` kwarg contains the symmetry operations that agree with
+the specified model (lattice, Hamiltonian terms etc.).
+By default the code checks the terms, atoms and magnetic moments
+to automatically determine a set of symmetries it can safely use.
+Use the `symmetry_operations` function if you want to pass custom symmetries
+(e.g. a reduced set) or force to employ all the symmetries returned by spglib.
+Notice that this may lead to wrong results if e.g. the external potential breaks
+symmetries. Alternatively setting this kwarg to `false` disables symmetries completely.
+In contrast setting it to `true` explicitly selects the default behaviour.
```

Comment:
This is a bit convoluted, there's no reason to support `true` anymore, right? Can't we just do `symmetries=nothing` (or `[]`) to mean what we use `false` for now?

Reply:
Yes, but it makes it more convenient to pass `false` to disable symmetries and `true` to get the default. E.g. for the tests where you just loop `for symmetries in `(true, false)` do.

Reply:
I reflexively don't like this sort of "user-friendly" APIs that are not really that convenient and that impose a maintenance burden later on. But OK here!

Reply:
I know I have a tendency to go that direction to early, but given the interface we had, i think this is a step up.